lib/index.js

```diff
@@ -26,6 +26,10 @@ EditionInterface.prototype.commit = require('./actions/commit.js');
 EditionInterface.prototype.commitFileAsync = require('./actions/commit-file-async.js');
 EditionInterface.prototype.dump = require('./actions/dump.js');
 
+const render = require('./actions/render.js');
+EditionInterface.prototype._render = render._render;
+EditionInterface.prototype._renderFile = render._renderFile;
```

Comment:
Since there's no dependencies on `this` in those functions; could we import them where they're used instead of exposing it on the prototype?

Reply:
Done, initially I wanted to allow overriding, it wasn't useful enough.
Now it's just forcing the cache, so this approach is better.